src/commands/config/eventxp/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Command } from "types/common"
+import { getCommandArguments } from "utils/commands"
+import { PREFIX } from "utils/constants"
+import { composeEmbedMessage, getErrorEmbed } from "utils/discordEmbed"
+import Config from "../../../adapters/config"
+
+const command: Command = {
+  id: "eventxp",
+  command: "eventxp",
+  brief:
+    "Toggle event XP. If enabled, users will get XP when they join the event.",
+  category: "Config",
+  onlyAdministrator: true,
+  run: async function(msg, action) {
+
+    const args = getCommandArguments(msg)
+    if (args.length != 2) {
+      return {
+        messageOptions: await this.getHelpMessage(msg)
+      }
+    }
+
+    const [toggleAction] = args.slice(1)
+		let active = false
+    switch (toggleAction.toLowerCase()) {
+			case "disable":
+				break
+			case "enable":
+				active = true
+				break
+			default:
+				return {
+					messageOptions: await this.getHelpMessage(msg)
+				}
+		}
```

Comment:
since this is a toggle comment, syntax should be `$eventxp` only.
enable it if currently disabled, and vice versa